ntbs-service/wwwroot/css/site.scss

```diff
@@ -140,4 +140,9 @@ body {
 
 .flex-container {
     display: flex;
+}
+
+.autocomplete__option {
+    color: black;
+    text-decoration: none;
```

Comment:
Overriding the libraries default behaviour - although strangely their examples don't have underlining and colour blue either...

Reply:
hmmm, yeah,i wonder if it's some interaction between our styles and theirs? or if we're missing some config..?
Not the end of the world if we have to stick with it, but perhaps own file with a small bit of explanation?

Reply:
I've put it into its own file now!